### DISCUSSION TITLE
Can logs be disabled for a single command?

### DISCUSSION THREAD

**Discussion by propri** [creativeprojects/resticprofile#358] [Q&A]
I just noticed resticprofile snapshots does not output to stdout if a log file is configured in global. The output is instead appended to the log.
This seems counterintuitive. I would expect snapshots to be output to stdout.
Is there an option or flag to output the result of certain commands to stdout?

**Comment by creativeprojects** [comment]
Hey!

Sorry I totally missed your query 😞 

Actually now there is a way: since the latest version `0.27.0`: you can use the flag `--command-output=console`.

There are also more options available as discussed here: https://github.com/creativeprojects/resticprofile/pull/343
